### PR TITLE
Test transaction with contract; pep8 fixes

### PR DESCRIPTION
**Pull request by panpilkarz**

### CODE REVIEW

File: pyethapp/tests/test_jsonrpc.py [hunk 1]
```diff
@@ -208,6 +210,38 @@ def test_send_transaction(test_app):
...
+    tx = {
+        'from': address_encoder(sender),
+        'to': address_encoder(tx_to),
+        'data': evm_code.encode('hex')
+    }
+    data_decoder(test_app.rpc_request('eth_sendTransaction', tx))
+    creates = chain.head_candidate.get_transaction(0).creates
+
+    code = chain.head_candidate.account_to_dict(creates)['code']
+    print code, `code`, type(code)
```
Comment (heikoheiko) [inline pyethapp/tests/test_jsonrpc.py]: nono

Comment (panpilkarz) [inline pyethapp/tests/test_jsonrpc.py]: damn. it is gone.

**Comment by panpilkarz** [comment]
up

**Comment by heikoheiko** [comment]
@ms83 shall this still be merged?

**Comment by panpilkarz** [comment]
Definitely. This PR contains a test, a bug fix and interface improvement.

**Pull request merged**